src/avs/AVSDetailsOperatorsTab.jsx

```
@@ -210,6 +213,20 @@ function AVSOperatorsList({ address, avsError, isAVSLoading, tvl }) {
     state.sort
   ]);
 
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await rdService.getAVSPromotedOperators(address);
+        dispatch({
+          promotedOperators: response.results,
+          promotedOperatorsRate: response.rate
+        });
+      } catch (e) {
+        //TODO: handle error
```

Why no error handling?

---

If the promoted operators endpoint fails, we don't need to display error, so we just swallow it up

---

That `TODO` confused me.

---

I'll reword this comment